### PR TITLE
Documentation for currency management

### PR DESCRIPTION
**Pull request by tasnuvacodemen**

### CODE REVIEW

File: docs/coin/coin-data-dictionary.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+cryptocurrency.
+
+## 3. Data Dictionary (CES V1.0)
+
+| Field Name                | Data Type    | Constraints                           | Description                                                                |
+| ------------------------- | ------------ | ------------------------------------- | -------------------------------------------------------------------------- |
+| id                        | UUID         | PRIMARY KEY, AUTO_INCREMENT, NOT NULL | Unique identifier for currency                                             |
+| symbol                    | VARCHAR(10)  | NOT NULL,UNIQUE                       | Unique identifier for currency                                             |
+| name                      | VARCHAR(50)  | NOT NULL                              | currency name                                                              |
+| coin_type                 | VARCHAR(20)  | NOT NULL                              | Type of currency                                                           |
```
Comment (devrob-go) [inline docs/coin/coin-data-dictionary.md:25]: should be tinyint with default 0

File: docs/coin/get-currency.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+```
+
+### Get Coin/Cuerrncy Response example:
+
+```json
+{
+   "Coin":{
+      "ID":"123e4567-e89b-12d3-a456-426614174000",
+      "Name":"Bitcoin",
+      "CoinType":"Crypto",
```
Comment (devrob-go) [inline docs/coin/get-currency.md:37]: make it uppercase.

File: docs/coin/get-currency.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+   "Symbol":"BTC",
+   "Name":"Bitcoin"
+}
+```
+
+### Get Coin/Cuerrncy Response example:
+
+```json
+{
+   "Coin":{
```
Comment (devrob-go) [inline docs/coin/get-currency.md:34]: Make it Data.

File: docs/coin/list-currency.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+   "StartDate":"2020-03-24T06:03:00.348+03:00",
+   "EndDate":""
+}
+```
+
+### List Coin/Currency Response example:
+
+```json
+{
+   "Coin":[
```
Comment (devrob-go) [inline docs/coin/list-currency.md:39]: Follow change request from first json.

File: docs/coin/update-currency.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+#### In cryptocurrency exchange system coin/currency can get details by ID or Symbol or Name.
+
+
+## 3. API
+
+### Get Coin/Cuerrncy Request example:
+
+```json
+{
+   "Coin":{
```
Comment (devrob-go) [inline docs/coin/update-currency.md:24]: Follow change request from first json.

File: docs/coin/get-currency.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+
+```json
+{
+   "ID":"123e4567-e89b-12d3-a456-426614174000",
+   "Symbol":"BTC",
+   "Name":"Bitcoin"
+}
+```
+
+### Get Coin/Cuerrncy Response example:
```
Comment (devrob-go) [inline docs/coin/get-currency.md:30]: Extend a section including Enum fields and string values (with uppercase)

**Review by devrob-go — CHANGES_REQUESTED** [review]

File: docs/coin/coin-data-dictionary.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+| decimals                  | INT          | NOT NULL                              | Number of decimal places for the currency                                  |
+| exchange_status           | tinyint(1)   | DEFAULT 1                             | Flag indicating whether currency exchange status active or not             |
+| deposit_status            | tinyint(1)   | DEFAULT 1                             | Flag indicating whether currency deposit status active or not              |
+| withdrawal_status         | tinyint(1)   | DEFAULT 1                             | Flag indicating whether currency withdrawal status active or not           |
+| is_active                 | tinyint(1)   | DEFAULT 1                             | Flag indicating whether the currency is active                             |
+| deposit_fee               | VARCHAR(50)  | NOT NULL                              | Value of the fee charged on each deposit                                   |
+| deposit_fee_type          | VARCHAR(20)  | NOT NULL,DEFAULT 'FEE_TYPE_FIXED'     | Type of fee charged on each deposit (e.g. fixed,percentage)                |
```
[the remark this answers is not in the document]
Comment (tasnuvacodemen) [reply docs/coin/coin-data-dictionary.md]: updated

File: docs/coin/coin-data-dictionary.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+| minimum_deposit_amount    | VARCHAR(50)  | NOT NULL                              | Minimum amount that can be deposited                                       |
+| maximum_deposit_amount    | VARCHAR(50)  | NOT NULL                              | Minimum amount that can be deposited                                       |
+| daily_deposit_limit       | VARCHAR(50)  | NOT NULL                              | Maximum amount that can be deposited in a day                              |
+| total_deposit             | VARCHAR(50)  | NOT NULL                              | Maximum amount of currency deposited                                       |
+| withdrawal_fee            | VARCHAR(50)  | NOT NULL                              | Value of the fee charged on each withdrawal                                |
+| withdrawal_fee_type       | VARCHAR(20)  | NOT NULL,DEFAULT 'FEE_TYPE_FIXED'     | Type of fee charged on each withdrawal(e.g. fixed, percentage)             |
```
[the remark this answers is not in the document]
Comment (tasnuvacodemen) [reply docs/coin/coin-data-dictionary.md]: updated

File: docs/coin/coin-data-dictionary.md (hunk 1 above)
Comment (tasnuvacodemen) [reply docs/coin/coin-data-dictionary.md]: updated

File: docs/coin/get-currency.md (hunk 2 above)
Comment (tasnuvacodemen) [reply docs/coin/get-currency.md]: updated

File: docs/coin/get-currency.md (hunk 3 above)
Comment (tasnuvacodemen) [reply docs/coin/get-currency.md]: updated

File: docs/coin/list-currency.md (hunk 4 above)
Comment (tasnuvacodemen) [reply docs/coin/list-currency.md]: updated

File: docs/coin/update-currency.md (hunk 5 above)
Comment (tasnuvacodemen) [reply docs/coin/update-currency.md]: updated

File: docs/coin/get-currency.md (hunk 6 above)
Comment (tasnuvacodemen) [reply docs/coin/get-currency.md:30]: updated

**Pull request merged**